metrics/metrics_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"testing"
+
+	. "github.com/pingcap/check"
+)
+
+func TestT(t *testing.T) {
+	TestingT(t)
+}
+
+var _ = Suite(&testSuite{})
+
+type testSuite struct {
+}
+
+func (s *testSuite) TestMetrics(c *C) {
+	PanicCounter.WithLabelValues(LabelDomain).Inc()
```

Comment:
What is the purpose of this test case?

Reply:
To test it doesn't panic when called.

Reply:
Please comment it. I am confused by this test.